src/components/tl/ucp/barrier/barrier_knomial.c

```diff
@@ -137,7 +137,12 @@ ucc_status_t ucc_tl_ucp_barrier_knomial_start(ucc_coll_task_t *coll_task)
     status = ucc_tl_ucp_barrier_knomial_progress(&task->super);
     if (UCC_INPROGRESS == status) {
         ucc_progress_enqueue(UCC_TL_UCP_TEAM_CORE_CTX(team)->pq, &task->super);
+<<<<<<< HEAD
```

Comment:
This is unresolved merge conflict - looks like you haven't tried to compile this.